lib/common/utils.c

```diff
@@ -1696,6 +1696,36 @@ attrd_update_delegate(crm_ipc_t * ipc, char command, const char *host, const cha
     return rc;
 }
 
+void
+handle_versioned_parameters(xmlNode *param_set, xmlNode *versioned_params)
+{
+    if (param_set && versioned_params) {
+        xmlNode *attrs = NULL;
+        xmlNode *attr = NULL;
+        xmlAttrPtr xIter = NULL;
+
+        for (attrs = __xml_first_child(versioned_params); attrs != NULL; attrs = __xml_next_element(attrs)) {
+            for (attr = __xml_first_child(attrs); attr != NULL; attr = __xml_next_element(attr)) {
+                const char *attr_name = crm_element_value(attr, XML_NVPAIR_ATTR_NAME);
+
+                if (attr_name && !crm_element_value(param_set, attr_name)) {
+                    crm_xml_add(param_set, attr_name, "null");
```

Comment:
This is indistinguishable from the literal string "null" used as a parameter value. Why is it necessary to have a value?